packages/test/test-end-to-end-tests/src/test/gc/gcTombstone.spec.ts

```diff
@@ -430,4 +430,53 @@ describeNoCompat("GC DataStore Tombstoned When It Is Sweep Ready", (getTestObjec
         assert(response.status === 404);
         assert(response.value.startsWith(`Datastore removed by gc`));
     });
+
+    // If this test starts failing due to runtime is closed errors try first adjusting `sweepTimeoutMs` above
+    itExpects("Can untombstone datastores by storing a handle",
+    [
+        { eventName: "fluid:telemetry:Summarizer:Running:InactiveObject_Loaded" },
+        {
+            error: "TombstonedDataStoreRequested",
+            eventName: "fluid:telemetry:ContainerRuntime:TombstonedDataStoreRequested",
+            viaHandle: true,
+        },
+    ],
+    async () => {
+        const {
+            unreferencedId,
+            summarizingContainer,
+            summarizer,
+        } = await summarizationWithUnreferencedDataStoreAfterTime(sweepTimeoutMs - waitLessThanSweepTimeoutMs);
+        const dataObject = await requestFluidObject<ITestDataObject>(summarizingContainer, unreferencedId);
+
+        // Wait enough time so that the datastore is sweep ready
+        await delay(waitLessThanSweepTimeoutMs);
```

Comment:
This is waiting for less than sweep timeout. How is the data store will be sweep ready after this?

Reply:
`summarizationWithUnreferencedDataStoreAfterTime(sweepTimeoutMs - waitLessThanSweepTimeoutMs)` creates an unreferenced datastore that is unreferenced for approximately (`sweepTimeoutMs - waitLessThanSweepTimeoutMs`), I'm thinking about renaming this, it's clear it's hard to read.

Reply:
Renamed to `remainingTimeUntilSweepMs` - this should be much easier to understand hopefully